test/tlc/test-yellow-taxi-trip.rb

```diff
@@ -0,0 +1,59 @@
+module TLC
```

Comment:
We don't need this module because we don't share our test and `YellowTaxiTripTest` will not be conflicted.

If we want to add `TLC` information to distinct from other test cases, `class TLCYellowTaxiTripTest` to reduce indent.

Reply:
> We don't need this module because we don't share our test and YellowTaxiTripTest will not be conflicted.

Thank you. I understood it. I will rename it to ` TLCYellowTaxiTripTest`🙏

Reply:
fix: [c75357d](https://github.com/red-data-tools/red-datasets/pull/151/commits/c75357d5141a1c4b73ee68b1c573d5e2b391d623)